ics/src/main/java/com/example/demo/controllers/ImagesController.java

```diff
@@ -64,7 +66,11 @@ public ResponseEntity<RecognitionResponseBody> classifyImage(@RequestBody Recogn
                 //todo nice to have by logic da poznae image-a, nqma da e ID
             }
 
-            imageRepository.saveAndFlush(image);
+            if(!noCache) imageRepository.saveAndFlush(image);
```

Comment:
It is considered bad practice to omit curly braces from if-else statements.

Reply:
adjusted